blocks/v2-section-header/v2-section-header.css

```diff
@@ -0,0 +1,55 @@
+.v2-section-header {
+  margin: 0 auto;
+  font-size: var(--heading-s-font-size);
+  line-height: var(--heading-s-line-height);
+  letter-spacing: var(--heading-s-letter-spacing);
+  text-align: center;
+  text-wrap: balance;
+}
+
+.v2-section-header > *:first-child {
+  margin-top: 0;
+}
+
+.v2-section-header > *:last-child {
+  margin-bottom: 0;
+}
+
+.v2-section-header__title {
+  font-size: var(--f-heading-4-font-size);
+  letter-spacing: var(--f-heading-4-letter-spacing);
+  line-height: var(--f-heading-4-line-height);
+}
+
+
+@media screen and (min-width: 1024px) {
```

Comment:
if the block is V2, then why do we need this breakpoint and old font variables? 🤔